crates/stark-felt/src/arbitrary.rs

```diff
@@ -0,0 +1,48 @@
+use lambdaworks_math::{field::element::FieldElement, unsigned_integer::element::UnsignedInteger};
+use proptest::prelude::*;
+
+use crate::Felt;
+const FIELD_HIGH: u128 = (1 << 123) + (17 << 64); // this is equal to 10633823966279327296825105735305134080
+const FIELD_LOW: u128 = 1;
+
+/// Returns a [`Strategy`] that generates any valid Felt
```

Comment:
Can you elaborate more on when we should use this function ? more generally when do we need a `Strategy`.